src/Sentiment/NewsHeadlineListener.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, symbol,url=None):
                                "polarity": polarity,
                                "subjectivity": subjectivity,
                                "sentiment": sentiment})
-                rds.set(md5Hash,True)
```

Comment:
The original console script store the article cache as Python list.  Since the python code exits once it's done fetching the articles in docker, it has to move to a 3rd party data store.